### PR TITLE
cypress: update help dialog screenshot automatically

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: I9f3e3c9cc7838f89d5a66a25c1f6dd0ef8f8a946

* Resolves: #3365 
* Target version: master 

### Summary
I have created two spec file to update the screenshot:
1) desktop/writer/help_dialog_update_spec.js
2) desktop/calc/help_dialog_update_spec.js

You can run this file individually using:
make UPDATE_SCREENSHOT=true check-desktop spec=writer/help_dialog_update_spec.js
If UPDATE_SCREENSHOT is true then  only cypress will run the spec file and replace the original screenshot

### CODE REVIEW

**Review by pedropintosilva — APPROVED** [review]

File: cypress_test/integration_tests/desktop/writer/help_dialog_update_spec.js [hunk 1]
```diff
@@ -0,0 +1,263 @@
+/* global describe it cy require afterEach beforeEach Cypress*/
+
+const { hideSidebar } = require('../../common/desktop_helper');
+var helper = require('../../common/helper');
+describe('Help dialog screenshot updation', function() {
```
Comment (kendy) [inline cypress_test/integration_tests/desktop/writer/help_dialog_update_spec.js:5]: Can you please add a bit of an explanation in form of a comment? :-)

File: cypress_test/plugins/index.js [hunk 2]
```diff
@@ -101,6 +101,11 @@ function pickTests(filename, foundTests) {
 			testsToRun = [];
 		}
 	}
+
+	if (!process.env.UPDATE_SCREENSHOT) {
```
Comment (kendy) [inline cypress_test/plugins/index.js:105]: Can we improve the condition a bit - it is hard to read: if (! something) { remove_something(); } ; would be preferred to have if (something) { add_something(); } - if possible...

**Review by kendy — COMMENTED** [review]
Looks good to be in general; some small changes would be good.

If possible - add also info how to trigger this to a README.

File: cypress_test/plugins/index.js (hunk 2 above)
Comment (Rash419) [reply cypress_test/plugins/index.js:105]: In this case, It is not possible

**Pull request merged**